lib/web/ui/mfa.go

```diff
@@ -51,3 +55,29 @@ func MakeMFADevices(devices []*types.MFADevice) []MFADevice {
 
 	return uiList
 }
+
+// MakeAuthenticateChallenge converts proto to JSON format.
+func MakeAuthenticateChallenge(protoChal *proto.MFAAuthenticateChallenge) *auth.MFAAuthenticateChallenge {
```

Comment:
Could we push this function somewhere where it may be reused outside of lib/web? There is code like this in various places, as you've seen.

Reply:
looked around and moved it to `lib/client/mfa.go`